projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/generated-methods-and-properties.migration.ts

```diff
@@ -4041,21 +4174,35 @@ export const GENERATED_METHODS_AND_PROPERTIES_MIGRATION: MethodPropertyDeprecati
       importPath: '@spartacus/storefront',
       deprecatedNode: 'generateJsonLdScript',
       comment:
-        "// TODO:Spartacus - The 'generateJsonLdScript' method's signature changed to: 'generateJsonLdScript(  schema: string | {}): SafeHtml | undefined'",
+        "// TODO:Spartacus - The 'generateJsonLdScript' method's signature changed to: 'generateJsonLdScript(  schema: string | {}): void'",
     },
     {
       class: 'JsonLdDirective',
       importPath: '@spartacus/storefront',
       deprecatedNode: 'jsonLD',
       comment:
-        "// TODO:Spartacus - The type of property 'jsonLD: SafeHtml' changed to: 'jsonLD: SafeHtml | undefined' ",
+        "// TODO:Spartacus - Property 'jsonLD' was removed from Class 'JsonLdDirective'. ",
+    },
+    {
+      class: 'JsonLdDirective',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'sanitizer',
+      comment:
+        "// TODO:Spartacus - Property 'sanitizer' was removed from Class 'JsonLdDirective'. ",
+    },
+    {
+      class: 'LaunchDialogService',
+      importPath: '@spartacus/storefront',
+      deprecatedNode: 'closeDialog',
+      comment:
+        "// TODO:Spartacus - The 'closeDialog' method's signature changed to: 'closeDialog(  reason: any): void'",
```

Comment:
reason is changed from `string` to `any`. I think it is not breaking change.

Reply:
I agree.  I'll remove it.